scripts/bundled_installer

```diff
@@ -55,6 +69,26 @@ function install_ebcli() {
     fi
 }
 
+function print_path_export_instructions__python() {
```

Comment:
Why not `export` the equivalent definitions in `python_installer` and simply use them here?

Reply:
Didn't realize you could export functions, that's a much cleaner way to do that. Will change.